src/linearmap.rs

```diff
@@ -318,10 +373,29 @@ mod tests {
         )
     }
 
+    #[test]
+    fn virtual_address_range_ttbr1() {
+        const GIB_512: isize = 512 * 1024 * 1024 * 1024;
```

Comment:
Extract to top level rather than repeating?

Reply:
Done.